.circleci/config.yml

```diff
@@ -59,6 +84,7 @@ workflows:
                 - main
           requires:
             - validate
+            - integration-test
             - build_docker
             - helm_lint
 #      - request-preprod-approval:
```

Comment:
This change adds a new, separate job in CircleCI for the gradle `integrationTest` target. It runs separately, and in parallel with, the `check` target, which runs `ktlintCheck` and `test` (unit) gradle targets under the hood.